src/components/ContactUs.js

```diff
@@ -1,72 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {
-    Button,
-    Form,
-    FormGroup,
-    Input,
-    Label,
-    Modal,
-    ModalBody,
-    ModalFooter,
-    ModalHeader
-} from 'reactstrap';
-
+import React, { Component } from 'react';
+import { Modal, Button } from 'antd';
 import { navLink, TEXTS } from './constants';
+import FillForm from './SmallerComponents/Form';
```

Comment:
there should not ba a `SmallerComponents` folder. This does not conform with our coding conventions